### ISSUE TITLE
/mv_26803389: The setting 'mvBlueFOX_SYS' does not exist at storage location 0x00000001 with scope 0x00000001

### ISSUE THREAD

**Issue by ChanghyeonKim93** [KumarRobotics/bluefox2#25]
Hello, I'm an user fo mvBlueFox mlc200wc camera.

Thanks to your instruction about how to install and run this camera with ROS, I can install the sdk and ROS wrapper easily. 

BUT, when I use the roslaunch bluefox2 single_node.launch devide:=26803389 (this is my device serial number) , it gives weird message like below : 

/mv_26803389: The setting 'mvBlueFOX_SYS' does not exist at storage location 0x00000001 with scope 0x00000001

and it is shutted down the very soon. 

I cannot find any issue about this error message ... Could someone kindly make me know how to fix this problem?

ps. the full script when roslaunch .

... logging to /home/chkim/.ros/log/607693fc-72ec-11e7-bd3a-1002b520b930/roslaunch-chkim-14ZD960-GX3GK-10246.log
Checking log directory for disk usage. This may take awhile.
Press Ctrl-C to interrupt
Done checking log file disk usage. Usage is <1GB.

started roslaunch server http://chkim-14ZD960-GX3GK:41632/

SUMMARY
========

PARAMETERS
 * /mv_26803389/aec: False
 * /mv_26803389/agc: False
 * /mv_26803389/calib_url: file://${ROS_HOME...
 * /mv_26803389/camera_name: mv_26803389
 * /mv_26803389/cbm: 0
 * /mv_26803389/ctm: 1
 * /mv_26803389/dcfm: 0
 * /mv_26803389/expose_us: 5000
 * /mv_26803389/fps: 60.0
 * /mv_26803389/frame_id: mv_26803389
 * /mv_26803389/gain_db: 0.0
 * /mv_26803389/hdr: False
 * /mv_26803389/identifier: 26803389
 * /mv_26803389/idpf: 0
 * /mv_26803389/image_raw/compressed/jpeg_quality: 100
 * /mv_26803389/mm: 0
 * /mv_26803389/request: 0
 * /mv_26803389/wbp: 1
 * /rosdistro: indigo
 * /rosversion: 1.11.21

NODES
  /
    mv_26803389 (bluefox2/bluefox2_single_node)

ROS_MASTER_URI=http://localhost:11311

core service [/rosout] found
process[mv_26803389-1]: started with pid [10264]
[ERROR] [1501175116.193683737]: /mv_26803389: The setting 'mvBlueFOX_SYS' does not exist at storage location 0x00000001 with scope 0x00000001
[mv_26803389-1] process has finished cleanly
log file: /home/chkim/.ros/log/607693fc-72ec-11e7-bd3a-1002b520b930/mv_26803389-1*.log
all processes on machine have died, roslaunch will exit
shutting down processing monitor...
... shutting down processing monitor complete
done

**Comment by versatran01** [comment]
Can you open the camera with wxPropView?

**Comment by ChanghyeonKim93** [comment]
yes, I can do in windows and Linux both.  So , I think that it is not a hardware problem.I found that 0x00000001 might be the error message related to driver . Do you have any idea about it? 

I' ve already recreate and rebuild my catkin workspace folder but I did not work.

**Comment by versatran01** [comment]
What camera are you using?

**Comment by ChanghyeonKim93** [comment]
Thank you for commenting. I use mlc200wc camera

**Comment by hmchung** [comment]
Hi @ChanghyeonKim93, not sure if you solved your issue. I had the same issue. Turns out that it is my faulty usb cable that caused the issue :(. By the way, also remember to give permission to your cameras either manually or by udev rules

**Comment by ChanghyeonKim93** [comment]
Hi @hmchung ! Thank you for answering. Actually, I already have found that I have to give 777 to my udev to use the cameras properly. (chmod 777 ) I guess it is not a default setting giving the permission to udev in Linux. Thanks~
